src/geom/Endpoint.cc

```diff
@@ -197,26 +197,26 @@ ndarray::Array<double, 2, 2> IcrsCoordEndpoint::dataFromArray(Array const& arr)
     return data;
 }
 
-coord::IcrsCoord IcrsCoordEndpoint::pointFromData(std::vector<double> const& data) const {
+SpherePoint SpherePointEndpoint::pointFromData(std::vector<double> const& data) const {
     this->_assertNAxes(this->_getNAxes(data));
-    return coord::IcrsCoord(data[0] * radians, data[1] * radians);
+    return SpherePoint(data[0] * radians, data[1] * radians);
```

Comment:
Any reason not to use the ```SpherePoint(data[0], data[1], radians)``` constructor here and below?

Reply:
Good idea. I'll do it.